test/alternator/test_gsi.py

```diff
@@ -1150,14 +1150,100 @@ def test_gsi_5_describe_table_schema(test_table_gsi_5):
     # The list of attribute definitions may be arbitrarily reordered
     assert multiset(got['AttributeDefinitions']) == multiset(expected_all_attribute_definitions)
 
+# A table for #5320 verification
+@pytest.fixture(scope="module")
```

Comment:
In https://github.com/scylladb/scylladb/pull/26047 I added a test for all (yes, all) possible combinations of base keys and GSI or LSI keys - 15 combinations for GSIs and 2 for LSIs, including a proof that these are ALL the combinations.

So if you agree with that, we don't need to add these 4 specific combinations.